tests/lib/devicetree/src/main.c

```diff
@@ -289,6 +289,17 @@ static void test_bus(void)
 		     "inst 0 i2c dev label");
 	zassert_true(!strncmp(i2c_bus, DT_INST_BUS_LABEL(0), strlen(i2c_bus)),
 		     "inst 0 i2c bus label");
+
+#undef DT_DRV_COMPAT
+	/*
+	 * Make sure the underlying DT_COMPAT_ON_BUS used by
+	 * DT_ANY_INST_ON_BUS works without DT_DRV_COMPAT defined.
+	 */
+	zassert_equal(DT_COMPAT_ON_BUS(vnd_spi_device, spi), 1, NULL);
```

Comment:
Should these be `zassert_true` and `zassert_false` instead?

Reply:
I'm using 1 and 0 because the API says those are the return values.

I want to make sure  it's not returning 2 or 3, etc.

Reply:
ok, we should do an audit of the other APIs/tests at some point.  I used zassert_true/zassert_false.

Reply:
OK. I didn't know that. I've tried to be pretty consistent about 0 and 1.